.rhcicd/clowdapp.yaml

```diff
@@ -129,19 +110,15 @@ parameters:
 - name: IMAGE_TAG
   description: Image tag
   value: latest
-- name: INFINISPAN_STORAGE
-  displayName: Infinispan's persistent storage size
-  description: Amount of storage to request for the database persistent volume
-  value: 10Gi
 - name: EXTERNAL_LOGGING_LEVEL
   value: INFO
 - name: MEMORY_LIMIT
   description: Memory limit
-  value: 3Gi
+  value: 1Gi
 - name: MEMORY_REQUEST
   description: Memory request
-  value: 1.5Gi
-- name: REPLICAS
+  value: 500Mi
+- name: MIN_REPLICAS
   value: "1"
```

Comment:
shouldn't this be something uneven? 3 at least?

Reply:
We usually have 1 replica in the ClowdApp template and that value is increased from app-interface depending on the target environment.

Reply:
ah. you are right. ok. then you get my lgtm